iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/aws/AwsWeakSSLProtocolCheckPart.java

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
If I understand correctly, you've effectively split the code of the old version of WeakSSLProtocolCheck
into this class (old-style checks) and then you've added class AzureWeakSSLProtocolCheckPart - new style checks?